router.lua

```diff
@@ -124,12 +125,20 @@ function Router:match(method, path, f)
   end
 end
 
-for method in ("get post put patch delete trace connect options head"):gmatch("%S+") do
+for _,method in ipairs(HTTP_METHODS) do
   Router[method] = function(self, path, f)     -- Router.get = function(self, path, f)
     return self:match(method:upper(), path, f) --   return self:match('GET', path, f)
   end                                          -- end
 end
 
+Router['any'] = function(self, path, f) -- match any method
+  local methodt = {}
+  for _,method in ipairs(HTTP_METHODS) do
+    methodt[method:upper()] = {[path] = f}
+  end
+  return self:match(methodt, path, f)
```

Comment:
* The `f` is not used, when table is passed
* I think it would be better, to wrap the `f` inside another function, that will pass the `method` instead of changing the `Router:execute`
* I'd rather use `Router:match` instead of crafting the table. it might be tiny bit less performant, but more clear

```lua
local method_table = {}
for _,method in ipairs(HTTP_METHODS) do
  self:match(method, path, function(params) return f(params, method) end)
end
```